portfolio/src/main/webapp/community.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Community Page</title>
+    <script src="community.js"></script>
+    <link rel="stylesheet" href="community.css">
+  </head>
+  <body onChange="renderComments()" onload="renderComments()">
+    <div id="community-content">
+        <div id="comment-form-section">
+            <div id="header">
+                <a href="index.html"> <img src=/images/profile.jpg /> </a>
+                <div class="left">
+                    <a href="about.html"> <p> About </p> </a>
+                    <a href="survey.html"> <p> Survey </p> </a>
+                    <a href="#"> <p> Contact </p> </a>
+                </div>
+            </div>
+            <h2>Here are all the wonderful things people have to say about this website</h2>
+            <form action="/comments" method="POST">
+                <p>Username:</p>
+                <input name="username" />
+
+                <p>Comment</p>
```

Comment:
Same comment about label instead of p as above =)

Reply:
Changed to label